pages/linux/wipefs.md

```diff
@@ -7,10 +7,14 @@
 
 `sudo wipefs {{/dev/sdX}}`
 
-- Wipe all available signatures for specified device:
+- Wipe all available signature types for a specific device (no recursion into partitions):
 
 `sudo wipefs --all {{/dev/sdX}}`
 
+- Wipe all available signature types for the device and partitions using a glob pattern:
+
+`sudo wipefs --all {{/dev/sdX*}}`
```

Comment:
Why does this need an asterisk, surely just giving the disk is enough if we are wiping all.

Feel free to ignore this, if it is not an error.

Reply:
The `*` is required, perhaps this better
```suggestion
`sudo wipefs --all {{/dev/sdX}}*`
```

Reply:
@CleanMachine1 

> surely just giving the disk is enough if we are wiping all

Fair question. It depends on intention.

- If you simply want to cleanup the device base signatures, yes, but all means "all signatures or signature types" at the block device level, and does not account for any signatures in other layers.
- If however, by "all" you want every signature across multiple layers, the glob is a partial hack to do that.
- E.g. /dev/sda may have two partions /dev/sda1 and /dev/sda2. If you wanted to wipe not just the partition table / label / signature, but also, filesystem signatures (or LVM, whatever is within the partitions), then the glob helps do that.
- Why? Well, besides a long zero erase, later on, if you use any data loss recovery tools that do a full block scan to find signatures, you ideally don't want any stale not-fully-erased signatures that could float in unallocated blocks read by the recovery program. It would just add to the confusion. E.g. imagine a bad program or something corrupts the partition table and it's backup, then the recovery program might scan the entire drive looking for filesystem signatures.